channels/client/addin.h

```diff
@@ -16,3 +16,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* we need rdpContext */
+//#include <freerdp/freerdp.h>
+
+void* channel_client_create_handler(rdpContext* ctx, LPVOID userdata,
+                                    UINT (*msg_handler)(LPVOID userdata, wStream* data),
+                                    const char* channel_name);
+
+UINT channel_client_post_message(void* MsgsHandle, LPVOID pData, UINT32 dataLength,
```

Comment:
Better use a specific typedef for the `MsgHandle`, so that the compiler can warn you when you pass in a wrong type (even if it is opaque)